docs/logs/extending-the-sdk/Program.cs

```diff
@@ -3,24 +3,25 @@
 
 using Microsoft.Extensions.Logging;
 using OpenTelemetry;
+using OpenTelemetry.Logs;
 
 namespace ExtendingTheSdk;
 
 public class Program
 {
     public static void Main()
     {
-        using var loggerFactory = LoggerFactory.Create(builder =>
-            builder.AddOpenTelemetry(options =>
-            {
-                options.IncludeScopes = true;
-                options.AddProcessor(new MyProcessor("ProcessorA"))
-                       .AddProcessor(new MyProcessor("ProcessorB"))
-                       .AddProcessor(new SimpleLogRecordExportProcessor(new MyExporter("ExporterX")))
-                       .AddMyExporter();
-            }));
-
-        var logger = loggerFactory.CreateLogger<Program>();
+        var sdk = OpenTelemetrySdk.Create(builder => builder
+            .WithLogging(logging =>
+            {
+                // logging.IncludeScopes = true;

```

Comment:
Help - I'm new, couldn't find this equivalent ...

Reply:
There's an overload to `WithLogging` that allows you to configure the options:

```csharp
var sdk = OpenTelemetrySdk.Create(builder => builder
    .WithLogging(
        logging => logging.AddConsoleExporter(),
        options => options.IncludeScopes = true));
```